zipkin-web/src/main/resources/app/js/component_data/default.js

```diff
@@ -1,15 +1,41 @@
 import {component} from 'flight';
 import $ from 'jquery';
+import queryString from 'query-string';
+import chai from 'chai';
```

Comment:
That's the whole point, it's going to get lost. (Chai will be removed as a dependency, only kept in tests.)
Commenting about Chai in the unit tests is somewhat redundant, since it's a core part of the node.js ecosystem. It's like explaining in src/test/scala what ScalaTest is.